apps/myaccount/src/api/validation.ts

```diff
@@ -135,3 +135,46 @@ export const getConfig = (ruleSet: ValidationConfInterface[], validator: string,
 
     return null;
 };
+
+/**
+ * Username validation configurations in the required format.
+ *
+ * @param configs - validation configurations for an organization.
+ * @returns the username validation configuration.
+ */
+export const getUsernameConfiguration = (configs: ValidationDataInterface[]): ValidationFormInterface => {
+
+    const usernameConf: ValidationDataInterface[] =
+        configs?.filter((data: ValidationDataInterface) => data.field === "username");
+
+    if (usernameConf === undefined || usernameConf.length < 1) {
+        return;
+    }
+
+    const config: ValidationDataInterface = usernameConf[0];
+    const rules: ValidationConfInterface[] = config?.rules;
+
+    if (rules?.length < 1) {
+        return;
+    }
+
+    return {
+        enableValidator: 
+                (
+                    getConfig(rules, "AlphanumericValidator", "enable.validator")=="true"
+                    || !(getConfig(rules, "EmailFormatValidator", "enable.validator")=="true")
+                )
+                    ? "true" 
+                    : "false",
+        field: "username",
+        maxLength: 
+            Number(getConfig(rules, "LengthValidator", "max.length"))
+                ? Number(getConfig(rules, "LengthValidator", "max.length"))
+                : null,
```

Comment:
```suggestion
            Number(getConfig(rules, "LengthValidator", "max.length")) ?? null,
```
We can minify like this. WDYT?

Reply:
Addressed with 2255de611fd68c6c9f2029e0bfd1dbab1c90e8c3.